### PR TITLE
Starting to add showPosition functionality.

### PR DESCRIPTION
**Pull request by D-Pagey**
It seems wrong to put in loads of functions within the Component. Should I just put `getLocation` and `showPosition` above the App component? It also seems like it would be wrong to have so much code in one file, especially at the Parent level.

### CODE REVIEW

**Comment by D-Pagey** [comment]
Also I could probably use ternary for the `getLocation` function rather than the `if/else`

File: src/App.js [hunk 1]
```diff
@@ -15,10 +15,27 @@ class App extends Component {
       max: '',
       min: '',
       description: '',
+      units: 'c',
       isLoading: false
     }
   }
 
+  showPosition(position) {
+    console.log(position);
```
Comment (iammatthewward) [inline src/App.js]: console.log

**Comment by iammatthewward** [comment]
it's not a problem to put functions inside of a component, but functions that would have a bit of a smell would be:

* functions that don't affect the view logic for that component
* functions that fetch data / calculate data that could otherwise be passed down as a prop

In the case of `getLocation`, you probably would want to pass the same function down as a callback, then pass the resulting position down as a prop. Then, your `componentDidMount` could be just updated to call `this.props.getLocation()` rather than `this.getLocation()`

**Comment by D-Pagey** [comment]
I don't understand this: 

> In the case of getLocation, you probably would want to pass the same function down as a callback, then pass the resulting position down as a prop. 

because this code is in App.js which is the top level component?

Actually, thinking about it. Are you saying I should make the function `getLocation` above this `App` component, and then pass it down to `<Data />` via props and call that function from inside `<Data />`?

**Pull request merged**